sycl/doc/GetStartedGuide.md

```diff
@@ -514,7 +514,8 @@ class CUDASelector : public cl::sycl::device_selector {
 
 ## C++ standard
 
-- Minimal supported C++ standard is C++11 on Linux and C++14 on Windows.
+- DPC++ runtime is built as C++14 library.
+- DPC++ compiler is building apps as C++14 apps by default.
```

Comment:
@mdtoguchi, is this true for the clang driver? Should we change the implementation to align it with this statement?

Reply:
Yes, this is true, both Host and Device compilations default to C++14.

Reply:
Right... I remember that we removed the custom logic setting C++11 some time ago.